doc/man7/EVP_PKEY-MLKEM.pod

```diff
@@ -0,0 +1,99 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY-MLKEM,
+EVP_PKEY-ML-KEM,
+EVP_KEYMGMT-ML-KEM,
+EVP_PKEY-ML-KEM-768
```

Comment:
Again the only valid algorithm name here is `ML-KEM-768`. You can't "fetch" anything else.

Reply:
Well, the name "EVP_PKEY-MLKEM" I only added after a test complained that it needs adding. The next two to allow the links from default provider to work -- exactly to avoid having to change more than 2 files as and when -512 and -1024 get added as per my initial comment.

Reply:
Changed equivalent as for PKEY above.

Reply:
You could try adding this somewhere:

```

=for openssl generic

```

(the empty lines surrounding it are significant)